### ISSUE TITLE
Non metric units?

### ISSUE THREAD

**Issue by jimbolaya** [nextcloud/weather#84]
I was hoping to see more non-metric units available than just temperature. 

Should I see an option under settings besides Metric with the temperature choice? 

This should probably be labelled as an Enhancement.

**Comment by e-alfred** [comment]
This is a duplicate of https://github.com/nextcloud/weather/issues/75. Currently you have to scroll down to the end of the page to see the unit selection. This wil be fixed with the next release.

**Comment by jimbolaya** [comment]
I'm sorry, I don't feel like this should be closed. #75 doesn't really address my issue. 

If temperature is the only unit to change, that's fine. I'm able to change it. However, the Pressure and Wind Speed units don't change and are still in metric. 

If you're not going to change them, that's fine, but please state that. If I should be able to change them, I cannot see any option to change units other than Temperature as the page only opens far enough for me to see Temperature options.

**Comment by e-alfred** [comment]
Yes, this would be a nice feature indeed. Which units do you think would be most useful?

**Comment by jimbolaya** [comment]
Well, being in the US, the units I see the most for air pressure are inches of mercury and wind speed would be miles per hour.

One weather app on my phone lists quite a few, but perhaps for wind speed one other that might be most useful would be knots.

Sorry about the initial lack of specificity in my original post.

**Comment by e-alfred** [comment]
Wind speed already changes to miles per second (it has the same abbreviation as meters per second `m/s`so only the values change), it just not indicated well in the drop down menu in the settings section.

Air pressure could be changed, I will take a look into it

**Comment by jimbolaya** [comment]
Great! Thank you.

As a note on wind speed, I'm not aware of anyone in the US, where I am, who uses miles per second as a wind speed unit for weather. As far as I know the normal units for wind speed are miles per hour.

**Comment by ldwellman** [comment]
I haven't seen any further comments on this thread, but I would also like to chime in and vote for it.  I'm in the US also, and I would love to see our imperial measurements on wind speed and barometric pressure.  As stated by @jimbolaya , our preference is mph (miles per hour) for wind speed and inches for barometric pressure.
Also, I believe the current showing of 'm/s' is NOT "miles per second", but is more likely "meters per second."  The numbers would be wildly improbable if miles per second.  My current weather shows 5.75 m/s.   If that were "miles," that would translate to 345 mph -- not likely on earth.  But if I consider it as 'meters per second,' that translates to about 13 mph.  The other local weather apps are showing local winds around 7 mph, so most likely, it's just a discrepancy due to time data was captured.

@e-alfred : I'm assuming you are in a metric-based country.  I ask this with sincerity and honesty -- Is it typical in your locale to express temperatures, wind speeds, and pressure to two decimal places?  For example, my local temperature is "40.15 F".  Even the Celsius value is "4.53."   Is that commonly done in metric countries?  If I could add another feature request, it would be to have the option to round these numbers to the nearest whole number.
